### PR TITLE
Add Login sample for SwiftUI

### PR DESCRIPTION
**Pull request by mrfour0004**
Add a sample project, `LineSDKSample-SwiftUI`, to demonstrate how to implement "Log in with LINE" feature on a SwiftUI project. Because the SDK doesn't seem to support SwiftUI officially, this pull request only focuses on the Login feature to help developers integrate LINE Login feature to their SwiftUI project.

The sample project copy `AppIcon` and `Config.xcconfig` from `LineSDKSample`, so the relevant settings are basically the same.

### CODE REVIEW

**Comment by onevcat** [comment]
截屏2022-07-04 9 37 14

@mrfour0004 

I am getting this when trying to build. Did you forget to add the async version of `logout` (which might be defined by you as an extension of `LoginManager`).

**Comment by mrfour0004** [comment]
> 截屏2022-07-04 9 37 14
> 
> @mrfour0004
> 
> I am getting this when trying to build. Did you forget to add the async version of `logout` (which might be defined by you as an extension of `LoginManager`).

Right, this file is missing while pushing, I've updated the commit, please check again 🙏🏻

File: LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//  IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+//  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import LineSDK
+import SwiftUI
+
+
+
```
Comment (onevcat) [inline LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/LoginView.swift:26]: Seems here is more space than other places. Please remove two empty lines.

File: LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/LoginView.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                }
+                .onLoginFail { error in
+                    presentedLoginError = error
+                }
+        }
+        .navigationBarTitle("Login", displayMode: .inline)
+        .alert(
+            "Success", isPresented: $presentedLoginResult, presenting: presentedLoginResult
+        ) { _ in
+            Button("Ok") { authorizationStore.isAuthorized = true }
```
Comment (onevcat) [inline LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/LoginView.swift:51]: Consider all capital case "OK" as it is standard on iOS platform.

File: LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/ProfileView.swift [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+struct ProfileView: View {
+    @EnvironmentObject private var authorizationStore: AuthorizationStore
+
+    @State var profile: UserProfile?
+
+    @State var logoutResult: Result<Void, LineSDKError>?
+
+    @State var alertMessage: String?
+
+//    @State var image
```
Comment (onevcat) [inline LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/ProfileView.swift:34]: Please remove this if it is not necessary.

File: LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/ProfileView.swift [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                case .failure(let error):
+                    alertMessage = error.localizedDescription
+                }
+            }
+        }
+    }
+
+    @ViewBuilder
+    private func profileContent(_ profile: UserProfile) -> some View {
+        if let url = profile.pictureURL, let data = try? Data(contentsOf: url) {
```
Comment (onevcat) [inline LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/ProfileView.swift:61]: Although it is should be fine to just block the thread by `Data(contentsOf:)` in a sample app, we are setting iOS 15.0 as our deployment target, so maybe consider `AsyncImage`?

Comment (mrfour0004) [reply LineSDKSample-SwiftUI/LineSDKSample-SwiftUI/Login/ProfileView.swift]: Great suggestion, deeply appreciated.

**Review by onevcat — APPROVED** [review]

**Comment by onevcat** [comment]
LGTM. It is so nice that now developers have a reference on using LINE SDK in a SwiftUI app.

Thanks for it!

**Pull request merged**